src/unstract/adapters/llm/open_ai/src/static/json_schema.json

```diff
@@ -24,11 +24,24 @@
       "default": 5,
       "description": "The maximum number of times to retry a request if it fails."
     },
+    "api_base": {
```

Comment:
NIT: @mohamed-siddhiq can you re-arrange the fields as below to have most relevant info on the top
1. adapter_name
1. api_key
1. model
1. api_base
1. api_version
1. max_retries
1. timeout

Reply:
@chandrasekharan-zipstack  The fields are re-arranged.